addresses/Addresses.json

```diff
@@ -52,6 +52,17 @@
         "name": "BRAVO_PROPOSER",
         "chainId": 31337,
         "isContract": false
-
+    },
+    {
+        "addr": "0x293cad26033577eb68137603a34d2dbfd05104d8",
+        "chainId": 11155111,
```

Comment:
why this chain?

Reply:
No specific reason. This is the ID for sepolia. I thought any EVM based chain would work.